plugin/flavor/zookeeper/flavor.go

```diff
@@ -167,7 +167,7 @@ func (s zkFlavor) Prepare(
 			return spec, errors.New("Manager nodes require an assigned logical ID")
 		}
 
-		spec.Init = generateInitScript(properties.UseDocker, allocation.LogicalIDs, *spec.LogicalID)
+		spec.Init += "\n" + generateInitScript(properties.UseDocker, allocation.LogicalIDs, *spec.LogicalID)
```

Comment:
Thanks for catching that, the newline should be conditional.  I meant to check if the init script introduced a shebang and it does, so the newline should only be included when appending.